packages/application-shell/README.md

```diff
@@ -72,3 +72,7 @@ ReactDOM.render(<EntryPoint />, document.getElementById('root'));
 | `render`                   | `func`   |    ✅    | -       | The function to render the application specific part. This function is executed only when the application specific part needs to be rendered. |
 | `trackingEventWhitelist`   | `object` |    ✅    | -       | An object containing a map of tracking events (_this mapping is required for backwards compatibility, it might be removed in the future_)     |
 | `onRegisterErrorListeners` | `func`   |    ✅    | -       | A callback function to setup global event listeners, called when the `ApplicationShell` is mounted                                            |
+
+## Testing
```

Comment:
`react-testing-library` is not limited to integration tests. It is also possible to write unit-tests with it in some situations. Generally, I've found that the distinction doesn't really matter much after switching to it.

And if you write unit-tests with another library you're back at the problem that the tests will break as soon as the implementation is changed, which is what we want to avoid. We want our tests to give us confidence while refactoring (changing the implementation). Coupling them to the implementation means they break when refactoring and are thus useless in this situation. Of course there are also unit-tests for pure function where the implementation can change.

I'm also hesitant of calling them "integration tests" because when somebody thinks about the testing pyramid, they would assume to write more unit tests than integration tests - but I would do the opposite. It should be more like a [testing trophy](https://twitter.com/kentcdodds/status/960723172591992832) instead. While this is not directly related, I think not labeling `react-testing-libarary` as integration-tests helps to avoid this wrong assumption.

`react-testing-library` never mentions "Integration testing" on its own Readme either.

That's why I'd want to avoid labeling `react-testing-library` as "integration tests", because I think that would set the expectations up wrong.

Hope this helped to explain where I'm coming from as this is not the first time `react-testing-library` has been stamped as "integration testing".

Reply:
I agree. I just meant that since our test utils are tightly coupled to react testing library, that we should mention that to avoid confusion.